zerver/tests/test_outgoing_webhook_system.py

```diff
@@ -190,6 +190,31 @@ def test_request_exception(self) -> None:
         assert bot_user.bot_owner is not None
         self.assertEqual(bot_owner_notification.recipient_id, bot_user.bot_owner.id)
 
+    def test_jsonable_exception(self) -> None:
+        bot_user = self.example_user('outgoing_webhook_bot')
+        mock_event = self.mock_event(bot_user)
+        service_handler = GenericOutgoingWebhookService("token", bot_user, "service")
+
+        response = ResponseMock(200, orjson.dumps(dict(content='whatever', widget_content={'test': 'test'})))
```

Comment:
Can you add a comment making clear what the problem is with the `widget_content` is?  I think this is an unexpected case -- we want `widget_content` to be some string that is not in JSON format, not an object of another type.

Longer-term we likely want to rewrite this test to use the `responses` library, like our authentication tests work, but we can leave that for a follow-up project.

Reply:
ok will do.

Reply:
I will follow up after this commit to look at how to rewrite the tests using `responses`